code/modules/admin/verbs/adminhelp.dm

```diff
@@ -354,8 +354,8 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		SEND_SOUND(initiator, sound('sound/effects/adminhelp.ogg'))
 
 		to_chat(initiator, "<font color='red' size='4'><b>- AdminHelp Rejected! -</b></font>")
-		to_chat(initiator, "<font color='red'><b>Your admin help was rejected.</b> The adminhelp verb has been returned to you so that you may try again.</font>")
-		to_chat(initiator, "Please try to be calm, clear, and descriptive in admin helps, do not assume the admin has seen any related events, and clearly state the names of anybody you are reporting.")
+		to_chat(initiator, "<font color='red'><b>Your admin help was rejected.</b> The adminhelp verb has been returned to you so that you may submit another ticket.</font>")
+		to_chat(initiator, "Please try to be calm, clear, and descriptive in admin helps. Assume that the admin has not seen any related events, and clearly state the character names or byond keys of anybody you are reporting.")

```

Comment:
This is literally the only change in this entire PR that'd actually be worth having.